binderhub/app.py

```diff
@@ -251,6 +251,51 @@ def _add_slash(self, proposal):
         """
     )
 
+    use_as_federation_portal = Bool(False, config=True,
+        help="""configure the current binder to also act as a federation portal.
+
+        This settings should be used only on mybinder.org deployment or if you
+        have multiple internal private deployment of binderhubs inside a closed
+        network. It exposes a couple of extra endpoints to binderhub for remote
```

Comment:
how about "It exposes extra endpoints on this instance that can be used by other binderhubs to register themselves"?